config/sync/core.extension.yml

```diff
@@ -36,6 +36,7 @@ module:
   page_cache: 0
   path: 0
   path_alias: 0
+  product_uploader: 0
```

Comment:
The product uploader module is enabled here. Though I don't see this module in this PR. Is it already in the codebase or it's here, by mistake? :)